lux/vislib/altair/SymbolMap.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright 2019-2020 The Lux Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from lux.vislib.altair.AltairChart import AltairChart
+import altair as alt
+import us
+import pycountry
+
+alt.data_transformers.disable_max_rows()
+
+
+class SymbolMap(AltairChart):
+    """
+    SymbolMap is a subclass of AltairChart that renders choropleth maps.
```

Comment:
Should we rename this GeoHeatmap or Choropleth now that we replaced SymbolMap with the heatmaps?

Reply:
Yup, changed this in the latest commit!